### PR TITLE
Update polkadot to v0.9.27

### PR DESCRIPTION
**Pull request by Niederb**

### CODE REVIEW

**Review by masapr — APPROVED** [review]
LGTM, but should not be merged like this (because it references the polkadot-branch of the pallets)

**Comment by Niederb** [comment]
> LGTM, but should not be merged like this (because it references the polkadot-branch of the pallets)

This is fixed now. Once the CI job passes this can also be merged

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -361,7 +361,7 @@ impl pallet_balances::Config for Runtime {
 	/// The ubiquitous event type.
 	type Event = Event;
 	type DustRemoval = ();
-	type ExistentialDeposit = ExistentialDeposit;
+	type ExistentialDeposit = ConstU128<EXISTENTIAL_DEPOSIT>;
```
Comment (clangenb) [inline runtime/src/lib.rs]: Is this really a necessary change?

**Review by clangenb — COMMENTED** [review]
I have a small remark, but it looks good otherwise!

File: runtime/src/lib.rs (hunk 1 above)
Comment (Niederb) [reply runtime/src/lib.rs]: I tried some a handful of approaches but struggled to get it to compile. In the end I copied the approach from the template:
https://github.com/substrate-developer-hub/substrate-node-template/blob/6a8b2b12371395979099d2c79ccc1860531b0449/runtime/src/lib.rs#L249
Do you have another option in mind?

Comment (clangenb) [reply runtime/src/lib.rs]: hmm, I don't know what happened to you. I double-checked and `cargo check -p integritee-node-runtime` does still compile without your change.

File: node/src/command.rs [hunk 2]
```diff
@@ -157,9 +157,23 @@ pub fn run() -> sc_cli::Result<()> {
...
+					},
+					BenchmarkCmd::Extrinsic(cmd) => {
+						let PartialComponents { client, .. } = service::new_partial(&config)?;
+						// Register the *Remark* and *TKA* builders.
+						let ext_factory = ExtrinsicFactory(vec![
+							Box::new(RemarkBuilder::new(client.clone())),
+							Box::new(TransferKeepAliveBuilder::new(
+								client.clone(),
+								Sr25519Keyring::Alice.to_account_id(),
+								EXISTENTIAL_DEPOSIT,
```
Comment (clangenb) [inline node/src/command.rs]: You can remove the workaround in the runtime by using here: `ExistentialDeposit::get()`

Comment (Niederb) [reply node/src/command.rs]: Ok, I implemented your suggestion

Comment (clangenb) [reply node/src/command.rs]: Thank you for your patience. :)

**Review by clangenb — APPROVED** [review]
Thanks for the fixes! Looks good to me now!

**Pull request merged**